patterns/call-to-action.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Title: Call to action
+ * Slug: twentytwentythree/cta
+ * Categories: featured
+ * Keywords: Call to action, signup, newsletter
+ * Block Types: core/buttons
+ */
+?>
+<!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide">
+	<!-- wp:column -->
+	<div class="wp-block-column">
+		<!-- wp:paragraph {"style":{"typography":{"lineHeight":"1.2"}},"fontSize":"x-large"} -->
+		<p class="has-x-large-font-size" style="line-height:1.2">Got any book recommendations?</p>
```

Comment:
Does this content need to be escaped/translated?

Reply:
Yes!